samples/javascript_nodejs/14.nlp-with-orchestrator/bots/dispatchBot.js

```diff
@@ -78,19 +79,19 @@ class DispatchBot extends ActivityHandler {
 
     async dispatchToTopIntentAsync(context, intent, recognizerResult) {
         switch (intent) {
-        case 'HomeAutomation':
-            await this.processHomeAutomation(context);
-            break;
-        case 'Weather':
-            await this.processWeather(context);
-            break;
-        case 'QnAMaker':
-            await this.processSampleQnA(context);
-            break;
-        default:
-            console.log(`Dispatch unrecognized intent: ${ intent }.`);
-            await context.sendActivity(`Dispatch unrecognized intent: ${ intent }.`);
-            break;
+            case 'HomeAutomation':
```

Comment:
undo whitespace changes

Reply:
Switch-case didn't have the correct spacing, so eslint extension took care of it.